app/client/components/payment/update/paymentUpdated.tsx

```diff
@@ -64,15 +68,23 @@ const ConfirmedNewPaymentDetailsRenderer = (subscription: Subscription) => {
   return <GenericErrorScreen loggingMessage="Unsupported new payment method" />; // unsupported operation currently
 };
 
-const WithSubscriptionRenderer = (withSub: WithSubscription) => (
+const WithSubscriptionRenderer = (productType: ProductType) => (
+  withSub: WithSubscription
+) => (
   <>
     <h1>Your payment details were updated successfully</h1>
     <ConfirmedNewPaymentDetailsRenderer {...withSub.subscription} />
     <h2>
-      Thank you. You are helping to support independent investigative journalism
+      Thank you. You are helping to support independent investigative
+      journalism.
     </h2>
     <div>
-      <LinkButton to="/" text="Manage your account" primary right />
+      <LinkButton
+        to={"/" + productType.urlPart}
+        text={"Manage your " + productType.friendlyName}
```

Comment:
Just an interesting one for the record: when we onboard Voucher and Home Delivery, they will be seen as one product/tab e.g. "Newspaper Subscription", with a page that must variate appropriately.

Reply:
noted thanks